org.simondalvai.ball2box.yml

```diff
@@ -0,0 +1,37 @@
+app-id: org.simondalvai.ball2box
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.godotengine.godot.BaseApp
+base-version: '3.5'
+command: godot-runner
+finish-args:
+  - --share=ipc
+  - --socket=x11 # <-- Godot needs X11
+  - --socket=pulseaudio
```

Comment:
I'd expect `--device=dri` to be needed for GL rendering.

Reply:
Great! This solves the performance issues I had.